### PR TITLE
Backport #9280

### PR DESCRIPTION
**Pull request by armandfardeau**

### CODE REVIEW

File: lib/decidim/budgets/component.rb [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    settings.attribute :vote_threshold_percent, type: :integer, default: 70
+    settings.attribute :vote_threshold_percent, type: :integer, default: 70
+    settings.attribute :vote_rule_minimum_budget_projects_enabled, type: :boolean, default: false
+    settings.attribute :vote_minimum_budget_projects_number, type: :integer, default: 1
+    settings.attribute :vote_rule_selected_projects_enabled, type: :boolean, default: false
+    settings.attribute :vote_selected_projects_minimum, type: :integer, default: 0
+    settings.attribute :vote_selected_projects_maximum, type: :integer, default: 1
+    settings.attribute :comments_enabled, type: :boolean, default: true
+    settings.attribute :comments_max_length, type: :integer, required: false
+    settings.attribute :geocoding_enabled, type: :boolean, default: false
```
Comment (paulinebessoles) [inline lib/decidim/budgets/component.rb:93]: I don't see the checkbox to enable the geocoding in the budget component creation or edit form

File: app/views/decidim/budgets/admin/projects/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<%= render partial: "decidim/shared/component_announcement" %>
```
Comment (paulinebessoles) [inline app/views/decidim/budgets/admin/projects/index.html.erb:1]: I think this file shouldn't be in the admin folder but in `app/views/decidim/budgets/projects/index.html.erb`

File: app/views/decidim/budgets/admin/projects/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,76 @@
+<% add_decidim_meta_tags(
```
Comment (paulinebessoles) [inline app/views/decidim/budgets/admin/projects/show.html.erb:1]: I think this file shouldn't be in the admin folder but in app/views/decidim/budgets/projects/index.html.erb

**Review by paulinebessoles — CHANGES_REQUESTED** [review]
Thank you @armandfardeau! I couldn't test because I couldn't find the checkbox in the budget component form to enable geocoding.
Could you please fix this?

File: app/views/decidim/budgets/admin/projects/index.html.erb (hunk 2 above)
Comment (armandfardeau) [reply app/views/decidim/budgets/admin/projects/index.html.erb:1]: Ok thx

**Comment by paulinebessoles** [comment]
Thanks @armandfardeau it works perfectly!

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**